authenticator/models.py

```diff
@@ -1,3 +1,78 @@
+#Django
 from django.db import models
+from django.contrib.auth.models import User
+
+class Personal_data(models.Model):
+
+    #Using User model from django
+    user = models.OneToOneField(User, on_delete=models.CASCADE)
+
+    #New fields to the personal_data table
+    country = models.CharField(max_length=20, blank=False)
+    city = models.CharField(max_length=15, blank=False) 
+    birth_date = models.DateField('%Y-%m-%d', blank=False)
+    
+    class gender(models.Model):
+        
+        FEMALE = 'F'
+        MALE = 'M'
+        NOT_IDENTIFIED = 'NI'
+        
+        GENDER_CHOICES = [
+            ( MALE, 'MALE'),
+            ( FEMALE, 'FEMALE'),
+            ( NOT_IDENTIFIED, 'NOT IDENTIFIED')
+        ]
+        gender = models.CharField(
+            
+            max_length=2,
+            choices= GENDER_CHOICES,
+            default= None
+        )         
+
+        def is_upperclass(self):
+            return self.gender in {self.MALE, self.FEMALE}
```

Comment:
Do a research about the Single Responsability Principle. Inner classes work, but are difficult to read and mantain, so define this `gender` class in another file and import it inside the `Personal_data` class